sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -1677,6 +1677,20 @@ class DataSourceV2SQLSuite
     }
   }
 
+  test("global temp view should not be masked by v2 catalog") {
```

Comment:
does the behavior make sense to you? @rdblue @brkyvz

Reply:
This is okay for now, but I think it is a little confusing that the catalog is completely ignored. I think this should result in an error instead, but we can do that in a follow-up.